src/cta/cta.py

```diff
@@ -53,11 +53,20 @@ def get_cta_message(data: pd.DataFrame) -> str:
     data = data[
         data['debt_to_supply_ratio'] >= 0.75
     ] 
+
+    if collateral_token_price is not None:
```

Comment:
Add a warning log here in case `collateral_token_price` is None pls 🙏🏼

Reply:
Added it in the `generate_and_store_ctas` function as logging it here would not provide any information about the address of the token for which we're missing a price. That would require additional argument for the `get_cta_message` and that seems kinda redundant.